autoload/neoterm/repl/python.vim

```diff
@@ -0,0 +1,57 @@
+"init python specific settings
+if !exists('g:neoterm_repl_ipy_magic')
```

Comment:
is it `magic` a term used in the python community? Is the name of this variable is meaningful to someone starting in the python community?

Reply:
Yes it is. [Read here](https://ipython.readthedocs.io/en/stable/interactive/magics.html). The currently used command use is the ipython-paste-magic.

Reply:
Great! That's good to know, thanks for sharing the reference. 
So, I think we should the _full qualified name_ here, just to be more clear:
```
if !exists('g:neoterm_repl_enable_ipython_paste_magic')
```
It's better to be a long name that it's more clear than a short one.